automation/TaskManager.py

```diff
@@ -15,9 +15,8 @@
 from .BrowserManager import Browser
 from .Commands.utils.webdriver_utils import parse_neterror
 from .CommandSequence import CommandSequence
-from .DataAggregator import BaseAggregator, LocalAggregator, S3Aggregator
-from .DataAggregator.BaseAggregator import (ACTION_TYPE_FINALIZE,
-                                            RECORD_TYPE_SPECIAL)
+from .data_aggregator import BaseAggregator, build_data_aggregator_class
+from .data_aggregator.base import ACTION_TYPE_FINALIZE, RECORD_TYPE_SPECIAL
```

Comment:
As an aside. @englehardt and I discussed incrementally moving the codebase to more conventional python naming (https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

Reply:
Does this mean, we'll also rename the base module from automation to openwpm?

Reply:
Not any time soon. But that seems logical to me.